src/payment/create-payment-strategy-registry.ts

```diff
@@ -341,6 +342,25 @@ export default function createPaymentStrategyRegistry(
         )
     );
 
+    registry.register(PaymentStrategyType.CHECKOUTCOM_GOOGLE_PAY, () => {
+        const initializer = new GooglePayCheckoutcomInitializer();
+
+        return new CheckoutcomPaymentStrategy(
```

Comment:
Because my response from googlepay its diferent from braintree or stripe, as they recibe a token as: 'tok_etcetc' and i receive several fields that must be send to checkout.com first an then checkout.com send a 'tok_etcetc', so the base generic implementation doesnt work for checkout.com